integration-libs/opf/checkout/styles/components/_opf-checkout-payments.scss

```diff
@@ -29,13 +30,6 @@
       background-color: var(--cx-color-light);
     }
 
-    > cx-opf-checkout-terms-and-conditions-alert {
```

Comment:
Aren't changes like this considered breaking changes?  I was under the impression that the CSS is all public interface.

Reply:
The same aspect as: https://github.com/SAP/spartacus/pull/20210/#discussion_r2079157877